app/views/UkimsNumberView.scala.html

```diff
@@ -0,0 +1,60 @@
+@*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import viewmodels.LabelSize._
+
+@this(
+    layout: templates.Layout,
+    heading1: components.Heading1,
+    paragraph: components.Paragraph,
+    formHelper: FormWithCSRF,
+    govukInput: GovukInput,
+    govukButton: GovukButton,
+    govukErrorSummary: GovukErrorSummary,
+    link: components.Link
+)
+
+@(form: Form[_])(implicit request: Request[_], messages: Messages)
+
+@layout(
+    pageTitle    = titleNoForm(messages("ukimsNumber.title")),
+    showBackLink = true
+) {
+    @heading1(messages("ukimsNumber.h1"))
+
+    @paragraph(Html(messages("ukimsNumber.p1")))
+    @paragraph(Html(messages("ukimsNumber.p2")))
+
+    @formHelper(action = routes.UkimsNumberController.onSubmit, Symbol("autoComplete") -> "off") {
+
+        @if(form.errors.nonEmpty) {
+            @govukErrorSummary(ErrorSummaryViewModel(form))
+        }
+
+        @govukInput(
+            InputViewModel(
+                    field = form("ukimsNumber"),
+                    label = LabelViewModel(messages("ukimsNumber.heading")).asPageHeading(Medium)
+            ).withHint(Hint(content = Text(messages("ukimsNumber.hint"))))
+        )
+
+        @govukButton(
+            ButtonViewModel(messages("site.continue"))
+        )
+    }
+
+    @paragraph(link(messages("ukimsNumber.linkText"), ""))
```

Comment:
Do we need to update this with a link at somepoint? Maybe a TODO?

Reply:
Good idea. Link updated